Jenkinsfile

```diff
@@ -18,12 +18,28 @@ pipeline {
 				sh 'git submodule update --init --recursive --remote'
 			}
 		}
+		stage('initialize PGP') {
+			steps {
+				withCredentials([file(credentialsId: 'secret-subkeys.asc', variable: 'KEYRING')]) {
+					sh 'gpg --batch --import "${KEYRING}"'
+					sh '''
+						for fpr in $(gpg --list-keys --with-colons | awk -F: \'/fpr:/ {print $10}\' | sort -u)
```

Comment:
@mickaelistria I tried to unroll the one-liner form TM4E https://github.com/eclipse/tm4e/blob/4e5ffed415a0b1a5504f42765beb803b4280a7c1/Jenkinsfile#L18
Is that correct?

Is it correct that this is to trust all keys in the ring?

Reply:
Yes, it's correct. The keyring is provided by the Eclipse Infra team, and what they provide can be trusted.
See https://wiki.eclipse.org/Jenkins#How_can_artifacts_be_deployed_to_OSSRH_.2F_Maven_Central.3F as reference documentation about PGP signatures at Eclipse.org.